packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -266,7 +267,7 @@ export default async (homeP, endowments) => {
     }
     if (!oracleArgs) throw Error(oraclesUsage);
     return Object.fromEntries(
-      oracleArgs.map(arg => {
+      JSON.parse(oracleArgs).map(arg => {
```

Comment:
as noted above if we're using `JSON.parse`, we might as well get rid of the regex and `Object.fromEntries`.